app/views/admin/accounts/_account.html.haml

```diff
@@ -23,13 +25,16 @@
             = relevant_account_timestamp(account)
             %small= t('accounts.last_active')
           %td.accounts-table__extra
-            - if account.local?
-              - if account.user_email
-                = link_to account.user_email.split('@').last, admin_accounts_path(email: "%@#{account.user_email.split('@').last}"), title: account.user_email
-              - else
-                \-
-              %br/
-              %samp.ellipsized-ip= relevant_account_ip(account, params[:ip])
+            - if current.role.permissions_as_keys.include?('sensitive_user_data') || current.role.permissions_as_keys.include?('administrator')
```

Comment:
Other code uses the `current_user.can?(:sensitive_user_data)` pattern (which makes me think that `:sensitive_user_data` could be called something like `:view_sensitive_user_data`)

This also avoids checking for the administrator permission since this implies all other permissions.

Reply:
Thanks for the feedback but I was not able to set up correctly the :view_sensitive_user_data that's why I took the other approach to make it work (i'm not very much into Ruby), any help would be appreciated.

Reply:
That's weird, I'd have expected `@current_user.can?(:sensitive_user_data)` to just work here.

Reply:
I'm trying your changes with an additional change of:

```suggestion
            - if @current_user.can?(:sensitive_user_data)
```

and it works as I'd expect

Reply:
Yeah you're right. I've mistook the @current_user with current_user so I was struggling. My bad.